artiq/compiler/embedding.py

```diff
@@ -88,18 +88,32 @@ def __init__(self, old_embedding_map=None):
                 self.subkernel_message_map[msg_type.name] = msg_id
                 self.object_reverse_map[obj_id] = msg_id
 
-        self.preallocate_runtime_exception_names(["RuntimeError",
-                                                  "RTIOUnderflow",
-                                                  "RTIOOverflow",
-                                                  "RTIODestinationUnreachable",
-                                                  "DMAError",
-                                                  "I2CError",
-                                                  "CacheError",
-                                                  "SPIError",
-                                                  "0:ZeroDivisionError",
-                                                  "0:IndexError",
-                                                  "UnwrapNoneError",
-                                                  "SubkernelError"])
+        # Keep this list of exceptions in sync with `EXCEPTION_ID_LOOKUP` in `artiq::firmware::ksupport::eh_artiq`
+        # The exceptions declared here should be defined in `artiq.coredevice.exceptions`
```

Comment:
should -> must. Should implies doesn't have to.

Reply:
Changed.